src/overlays/actors/ovl_Bg_Ctower_Gear/z_bg_ctower_gear.c

```diff
@@ -198,18 +178,19 @@ void BgCtowerGear_UpdateOrgan(Actor* thisx, GlobalContext* globalCtx) {
     }
 }
 
-// Using BgCtowerGear *this = THIS causes regalloc issues
 void BgCtowerGear_Draw(Actor* thisx, GlobalContext* globalCtx) {
-    func_800BDFC0(globalCtx, D_80AD32E8[BGCTOWERGEAR_GET_TYPE(thisx)]);
+    static Gfx* sDLists[] = { gClockTowerCeilingCogDL, gClockTowerCenterCogDL, gClockTowerWaterWheelDL };
+
+    func_800BDFC0(globalCtx, sDLists[BGCTOWERGEAR_GET_TYPE(thisx)]);
```

Comment:
I'm not a huge fan of having in-function static data if it isn't required to match. I would personally prefer to put it outside of the function, but up to you.